src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADAppCredentialCommand.cs

```diff
@@ -72,8 +74,9 @@ public override void ExecuteCmdlet()
                     ObjectId = ActiveDirectoryClient.GetObjectIdFromApplicationId(ApplicationId);
                 }
 
+                string decodedPassword = Utilities.SecureStringToString(Password);
```

Comment:
Please use [this ](https://github.com/Azure/azure-powershell/blob/77afe7be817242e61026069d2e8d3befe7984427/src/Common/Commands.Common/Extensions/SecureStringExtensions.cs) conversion method instead (and change in other files).